auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -799,6 +803,11 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
     await this.syncBusinesses()
   }
 
+  async popupBusinessDialog(business:Business) {
+    this.initialBusinessIdentifierManageBusinessDialog = business.businessIdentifier
```

Comment:
possible to just use this.businessIdentifier?

Reply:
if there are multiple affiliation requests for different business in the same account it will have multiple possible business identifiers. So I am passing this one to open it for specific (this is 2ndary action button on affiliation table)

Reply:
Shouldn't you be setting a list then?

Reply:
Business Dialog can only handle 1 business identifier at a time I think?

Reply:
Yeah, but this variable is used just for starting it up, and is defined in the EntityManagement vue. Where there might be multiple businessIdentifiers. But sure, I can rename it to businessIdentifier